spec/features/editing_file_attachments/upload_file_attachment_spec.rb

```diff
@@ -30,15 +30,25 @@ def and_i_upload_a_file_attachment
     @asset_manager_request = stub_asset_manager_receives_an_asset(filename: @attachment_filename)
     @publishing_api_request = stub_publishing_api_put_content(@edition.content_id, {})
 
-    @attachment_filename = "attachment.csv"
+    @attachment_filename = "attachment.pdf"
     find('form input[type="file"]').set(Rails.root.join(file_fixture(@attachment_filename)))
     fill_in "title", with: "A title"
     click_on "Upload"
   end
 
   def then_i_can_see_the_attachment_markdown
-    expect(page).to have_content("[Attachment: attachment.csv]")
-    expect(page).to have_content("[AttachmentLink: attachment.csv]")
+    expect(page).to have_content("[Attachment: attachment.pdf]")
+  end
+
+  def and_i_can_see_a_preview_of_the_attachment_link
```

Comment:
Also noticed that this method isn't actually called.

Reply:
D'oh - added to the scenario block.